deploy/charts/cert-manager/templates/webhook-deployment.yaml

```diff
@@ -44,7 +44,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       {{- if and .Values.prometheus.enabled (not (or .Values.prometheus.servicemonitor.enabled .Values.prometheus.podmonitor.enabled)) }}
-      {{- if not .Values.podAnnotations }}
+      {{- if not .Values.webhook.podAnnotations }}
```

Comment:
This is a drive-by fix for a mistake I made in https://github.com/cert-manager/cert-manager/pull/7182
 * https://github.com/cert-manager/cert-manager/pull/7182